pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -31,15 +31,16 @@ import (
 var logger = log.New("aries-framework/controller/did-exchange")
 
 const (
-	operationID             = "/connections"
-	createInvitationPath    = operationID + "/create-invitation"
-	receiveInvitationPath   = operationID + "/receive-invitation"
-	acceptInvitationPath    = operationID + "/{id}/accept-invitation"
-	connections             = operationID
-	connectionsByID         = operationID + "/{id}"
-	acceptExchangeRequest   = operationID + "/{id}/accept-request"
-	removeConnection        = operationID + "/{id}/remove"
-	connectionsWebhookTopic = "connections"
+	operationID                  = "/connections"
+	createInvitationPath         = operationID + "/create-invitation"
+	createImplicitInvitationPath = operationID + "/create-implicit-invitation"
```

Comment:
we only added "/create-implicit-invitation" but its showing all the const are re added. May be extra space issue

Reply:
go format